WordPress/Classes/Extensions/AbstractPost+PostInformation.swift

```diff
@@ -16,4 +16,9 @@ extension AbstractPost: ImageSourceInformation {
     var isLocalDraft: Bool {
         return self.isDraft() && !self.hasRemote()
     }
+
+    /// An autosave revision may include post title, content and/or excerpt.
+    var isAutosaveRevisionAvailable: Bool {
+        return autosaveModifiedDate != nil
```

Comment:
This is based on a (risky) assumption that if the autosave date is `nil` there is definitely no autosave available. Even if this assumption is true, I should be checking for `NSNull`. 

I can't use content, title and excerpt to validate if there is or isn't an autosave, because a valid autosave can occur when all these properties are null I suppose. I'll give this more thought.

Reply:
Android uses the [existence of the revision id](https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/develop/fluxc/src/main/java/org/wordpress/android/fluxc/model/PostModel.java#L398-L401). It seems safer. Perhaps we can use that?

Reply:
That's a good idea, I hadn't considered all the parameters unfortunately. The logic will then be: if the revision id is not `nil` nor `NSNull`, and is greater than zero - there is an autosave available.